plugins/reporters/evaluated-model/src/main/kotlin/EvaluatedModelMapper.kt

```diff
@@ -635,8 +636,11 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
     ) {
         val pathExcludes = getPathExcludes(id, scanResult.provenance)
         val licenseFindingCurations = getLicenseFindingCurations(id, scanResult.provenance)
+        // TODO: Decide whether the ordering in the output should be independent of the order of the input and
+        //       either remove below sortedWith() call, or replace it with a proper ordering mechanism for the whole
+        //       model.
```

Comment:
Given that the evaluated model report is a bit special in the sense that it tries to "pre-compute" as much as possible for quick display in the web-app report, I think it's fair to stick sorting here (if that's the way things should be show in the web-app report).

Reply:
So maybe replace the `TODO` here with a comment that explains that sorting is done here to avoid the need to sort in the web-app each time the report is loaded.